keps/sig-cli/4292-kubectl-debug-custom-profile/README.md

```diff
@@ -957,6 +957,7 @@ Major milestones might include:
 -->
 - 2023-10-13: Kep is proposed as alpha feature
 - 2024-06-04: Kep is promoted to beta
+- 2024-09-05: Kep is promoted to stable
```

Comment:
I'll drop comments here, b/c I can't do that in not-modified parts:
- in `##### Unit tests` section please update the coverage after the addition of the new test cases described there. 
- in `##### Integration tests` link k8s-triage pointing to new tests.
- in `##### e2e tests` add links to newly added e2e, if there are some 
- in `#### GA` section, as mentioned below, we should state that the feature gate will be locked to true, and removed in n+2 releases, to follow what we're doing in case of regular feature gates, which we should follow as closely as possible.
- Missing answer for 2 questions `###### What are other known failure modes?` and `###### What steps should be taken if SLOs are not being met to determine the problem?`. The first deserves an answer, the latter can be marked as NA.
-

Reply:
> in ##### Unit tests section please update the coverage after the addition of the new test cases described there.

Added latest 67.3% test coverage.

> in ##### Integration tests link k8s-triage pointing to new tests.

Added. However, debug.sh tests are not shown in this link, so I had to give the integration test link.

> in ##### e2e tests add links to newly added e2e, if there are some

This PR https://github.com/kubernetes/kubernetes/pull/127187 will add this

> in #### GA section, as mentioned below, we should state that the feature gate will be locked to true, and removed in n+2 releases, to follow what we're doing in case of regular feature gates, which we should follow as closely as possible.

That is good point, I tried to reflect this.

> Missing answer for 2 questions ###### What are other known failure modes? and ###### What steps should be taken if SLOs are not being met to determine the problem?. The first deserves an answer, the latter can be marked as NA.

Tried to answer.

Reply:
@soltysh I incorporated your suggestions. Please let me know what you think about the latest version. Thank you.